certified-connectors/Knowledgeone RecFind 6/apiDefinition.swagger.json

```diff
@@ -0,0 +1,376 @@
+          {
+            "name": "SearchText",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "description": "Search Text",
+            "x-ms-summary": "Text to filter the search results"
+          }
+        ]
+      }
+    },
+    "/SendFile": {
+      "post": {
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "URL": {
+                  "type": "string",
+                  "description": "URL"
+                },
+                "result": {
+                  "type": "string",
+                  "description": "result"
+                }
+              }
+            }
+          }
+        },
+        "summary": "Send File",
+        "description": "Submit file contents and metadata for storage in the RecFind 6 database. Returns a URL to the stored file.",
+        "operationId": "SendFile",
+        "x-ms-visibility": "important",
+        "parameters": [
+          {
+            "name": "HostUrl",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "x-ms-summary": "Host URL",
+            "description": "URL of the server"
+          },
+          {
+            "name": "UserName",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "x-ms-summary": "User Name",
+            "description": "User making the request"
+          },
+          {
+            "name": "body",
+            "in": "body",
+            "required": false,
+            "schema": {
+              "type": "object",
+              "properties": {
+                "FileContents": {
+                  "type": "string",
+                  "description": "FileContents"
+                },
+                "FileName": {
+                  "type": "string",
+                  "description": "FileName"
+                },
+                "Title": {
+                  "type": "string",
+                  "description": "Title"
+                },
+                "CreatedDate": {
+                  "type": "string",
+                  "description": "CreatedDate"
+                },
+                "EDOCType": {
+                  "type": "string",
+                  "description": "EDOCType"
+                },
+                "ExtraFields": {
+                  "type": "array",
+                  "items": {
+                    "type": "object",
+                    "properties": {
+                      "FldName": {
+                        "type": "string",
+                        "description": "FldName"
+                      },
+                      "FldValue": {
+                        "type": "string",
+                        "description": "FldValue"
+                      }
+                    }
+                  },
+                  "description": "ExtraFields"
```

Comment:
@ecardakaris-knowledgeonecorp All the description values of properties are expected to be free from grammatical and spell errors. Please provide valid description values.
Docs - [Define operation and parameter summaries and descriptions](https://docs.microsoft.com/connectors/custom-connectors/certification-submission#define-operation-and-parameter-summaries-and-descriptions)

Applies the same comment to all the similarly property values.